File: 1051. Height Checker/1051.go
package leetcode

import "sort"

func heightChecker(heights []int) int {
	heightsSort := make([]int, len(heights))
	copy(heightsSort, heights)
	sort.Ints(heightsSort)
	var count int
	for i, v := range heights{
		if v != heightsSort[i]{
			count ++
		}
	}
	return count
}

File: 1051. Height Checker/1051.py
class Solution:
    def heightChecker(self, heights) -> int:
        import copy
        heightsSort = copy.copy(heights)
        heightsSort.sort()
        count = 0
        for i in range(len(heights)):
            if heights[i] != heightsSort[i]:
                count += 1
        return count

a = Solution()
print(a.heightChecker([1,1,4,2,1,3]))